docs/content/en/docs/implementing/metrics.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Keptn Metrics
+description: Implement Keptn site metrics
+weight: 130
+---
+
+Kubernetes provides two metrics servers,
+one that is custom, the other external.
+However, the Kubernetes metrics servers
+only allow you to release a single service on a API;
+you cannot use multiple observability platforms
+in one namespace for one API.
+
+The Keptn Metrics Operator provides a single entry point to all metrics in the cluster
+and allows you to use multiple observability platforms
+and, beginning with v0.8.0,
+multiple instances of any observability platform.
+Keptn Metrics are also application aware
+so report data for all workflows that are included in the Keptn application.
```

Comment:
```suggestion
so they report data for all workflows that are included in the Keptn application.
```
Does this sound better?  What was there was actually a complete sentence but perhaps this is clearer.